### DISCUSSION TITLE
Add advanced tutorial of [customize transforms] 文档撰写：进阶教程-自定义数据增强

### DISCUSSION THREAD

**Discussion by OpenMMLab-Assistant-004** [open-mmlab/OpenMMLabCamp#259] [MMPose]
# Add advanced tutorial of [customize transforms] 
 文档撰写：进阶教程-自定义数据增强 

Refer to [doc]([https://mmengine.readthedocs.io/en/latest/advanced_tutorials/data_transform.html)](https://mmengine.readthedocs.io/en/latest/advanced_tutorials/data_transform.html))  to add a doc of Advanced Guide-Customize Data Transforms:
 1. Provide an example of customizing transform. Taking a MMPose provided transform as example is also acceptable.
 2. Explain details of inputs and outputs of transform(). Help users to understand how to create a new transform.
 3. Add both of Chinese and English version docs. Save as docs/en/advanced_guides/customize_transforms.md and docs/zh_cn/advanced_guides/customize_transforms.md

参考 [doc]([https://mmengine.readthedocs.io/en/latest/advanced_tutorials/data_transform.html)](https://mmengine.readthedocs.io/en/latest/advanced_tutorials/data_transform.html))  为 MMPose 撰写AdvancedGuide-Customize Data Transform文档：
 1. 提供一个自定义 transform 的示例，也可以用 MMPose 中已有的数据增强作为例子
 2. 需要讲解清楚transform()的输入输出，用户如何创建新数据增强
 3. 提供中英文，内容分别存放到docs/en/advanced_guides/customize_transforms.md和docs/zh_cn/advanced_guides/customize_transforms.md 中

## Task Information

**Repository**：MMPose

**Difficulty**：Easy

**Points**：20

**Estimated Time Consumption**：14 days

## Task Status

**Claimant's Github ID**：chaodyna

**Claim Time**：2023年07月24日

**PR Link**：https://github.com/open-mmlab/mmpose/pull/2605

## Points to Note

❗️请注意：此任务属于超级视客营任务，不能被领取！请登陆 [超级视客营平台](https://openmmlab.com/activity/codecamp) 按照流程申请任务，我们会尽快审核～

**Comment by lawenlic** [comment]
I will try it.
